actpool/actqueue.go

```diff
@@ -86,6 +88,8 @@ type actQueue struct {
 	pendingBalance *big.Int
 	clock          clock.Clock
 	ttl            time.Duration
+
+	mu sync.RWMutex
```

Comment:
as we chatted, we don't need this? 8 threads, each dealing with 1/8 of address space, each thread has a `map[sender]actQueue`, only 1 thread will operate 1 particular actQueue

Reply:
yes. It's necessary